blackbox/docs/general/builtins/scalar.rst

```diff
@@ -734,6 +734,57 @@ The ``timezone`` will be ``UTC`` if not provided::
     +------------------+
     SELECT 1 row in set (... sec)
 
+``timezone(timezone, timestamp)``
+---------------------------------
+
+The timezone scalar function converts values of ``timestamp`` without time zone to/from
+timestamp with time zone.
+
+Synopsis
+........
+
+::
+
+    TIMEZONE(timezone, timestamp)
+
+It has two variants depending on the type of ``timestamp``:
+
+.. csv-table::
+   :header: "Type of timestamp", "Return Type", "Description"
+
+   timestamp without time zone OR bigint, timestamp with time zone, Treat given timestamp without time zone as located in the specified timezone
+   timestamp with time zone, timestamp without time zone, Convert given timestamp with time zone to the new timezone with no time zone designation
+
+::
+
+    cr> select timezone('Europe/Madrid', 257504400000) as epoque;
```

Comment:
Maybe we could change those to also include a formatted variant.

```
select timezone('Europe/Madrid', 257504400000) as epoque,
          date_format('%Y-%m-%d %h:%i', timezone(...
```

Given that the time in ms is hard to parse.

Reply:
done